src/register/RegistrationPage.jsx

```diff
@@ -653,12 +653,16 @@ class RegistrationPage extends React.Component {
                   referrer={REGISTER_PAGE}
                 />
               )}
-              <div className="text-left">
-                <span className="mr-1">{intl.formatMessage(messages['already.have.an.edx.account'])}</span>
-                <a href={LOGIN_PAGE} onClick={this.handleLoginLinkClickEvent}>{intl.formatMessage(messages['sign.in.hyperlink'])}</a>
+              <div className="">
+                <p>
```

Comment:
do we need the <div> tag above? or just <p> is enough

Reply:
I guess no, let me remove it.